include/deal.II/base/partitioner.h

```diff
@@ -282,6 +318,67 @@ namespace Utilities
        */
       bool ghost_indices_initialized() const;
 
+#ifdef DEAL_II_WITH_MPI
+      /**
+       * Starts the exports of the data in a locally owned array to the range
+       * described by the ghost indices of this class.
```

Comment:
can you please enlarge the documentation to explain what, say, `temporary_storage` is for and what is `ghost_array`, etc? That's is not obvious (for me) from the current description. Or maybe extend the class documentation with a quick overview of the communication algorithm to motivate those variables.

Reply:
I added an extensive documentation of the send/receive model in the general class description and a documentation of all parameters to the given functions.

Reply:
thanks you!